feat: DIA-1969: Support MIG Image use case

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/13900146057)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/423
<!-- Follow Merge -->

niklub committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13566841395

niklub committed:
Merge branch 'develop' into 'fb-dia-1969'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13607399335

niklub committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13607399335

niklub commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13715789975)
> Successfully merged:  137 files changed, 6305 insertions(+), 677 deletions(-)

niklub committed:
Merge branch 'develop' into 'fb-dia-1969'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13715774766

niklub committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13715774766

niklub commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13787235735)
> Successfully merged:  create mode 100644 web/libs/ui/src/lib/enterprise-badge/enterprise-badge.tsx

niklub committed:
Merge branch 'develop' into 'fb-dia-1969'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13787235735

niklub commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13833975168)
> Successfully merged:  47 files changed, 1307 insertions(+), 213 deletions(-)

niklub committed:
Merge branch 'develop' into 'fb-dia-1969'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13833975168

niklub committed:
Merge branch 'develop' into 'fb-dia-1969'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13859408553

niklub committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13859408553

niklub committed:
Merge branch 'develop' into 'fb-dia-1969'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13900146057

niklub committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13900146057
